### PR TITLE
Cleanup ProgressBar code

### PR DESCRIPTION
**Pull request by hughbe**
## Proposed Changes
- Rename `ProgressBar` fields
- Follow coding guidelines in `ProgressBar`

Tested by #2141 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2169)

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/ProgressBar.cs [hunk 1]
```diff
@@ -82,202 +63,143 @@ protected override CreateParams CreateParams
...
                 }
+
                 return cp;
             }
         }
 
-        [Browsable(false), EditorBrowsable(EditorBrowsableState.Never)]
+        [Browsable(false)]
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public override bool AllowDrop
```
Comment (zsd4yr) [inline src/System.Windows.Forms/src/System/Windows/Forms/ProgressBar.cs]: why do these overrides exist at all?

Comment (hughbe) [reply src/System.Windows.Forms/src/System/Windows/Forms/ProgressBar.cs]: To modify the visibility of the event for editors

File: src/System.Windows.Forms/src/System/Windows/Forms/ProgressBar.cs [hunk 2]
```diff
@@ -793,23 +668,25 @@ internal override bool ShouldSerializeForeColor()
         public override string ToString()
         {
             string s = base.ToString();
-            return s + ", Minimum: " + Minimum.ToString(CultureInfo.CurrentCulture) + ", Maximum: " + Maximum.ToString(CultureInfo.CurrentCulture) + ", Value: " + value;
+            return s + ", Minimum: " + Minimum + ", Maximum: " + Maximum + ", Value: " + _value;
```
Comment (JeremyKuhne) [inline src/System.Windows.Forms/src/System/Windows/Forms/ProgressBar.cs]: nit: can you change this to an interpolated string? i.e. `$"{base.ToString()}, Minimum: {Minimum}..."`

Comment (hughbe) [reply src/System.Windows.Forms/src/System/Windows/Forms/ProgressBar.cs]: Done!

**Pull request merged**